index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body> 
+  
+  <h1>Kooking in the Kitchen: Women Redesigning Design</h1> 
+  
+  <ul>
+    <li>ABOUT</li>
+      <p>
+        Attention! Anyone sick of learning about the white male big shots that dominate design industries. 
+        Woman-identifying artists tell their stories and address how navigate male-dominated design fields. 
+        Sponsored by the KITCHEN KONVERSATIONS podcast, Faith Shakira and Rebecca Loren have created this 
+        event for women to claim a rightful place within creative industries. Join us for the first annual 
+        Kooking in the Kitchen speaker series. 
+      </p> 
+      <p>
+        For more information, feel free to visit <a href="https://open.spotify.com/show/2GEnLE5IrXKWew4jLXFLxV">KITCHEN KONVERSATIONS</a> on Spotify.
+    </p>
+    
+    <li>TALKS</li>
+      <ul>
+        <li>DESIGNING CULTURE 05/20 18:00
+          <p>Ji Won Choi and Solange discuss where culture falls within design, and how to navigate 
+          intersections between "design culture" and "experiential culture."
+          </p></li>
+        <li>OTHER THAN THE WHITE MALE IN THE ROOM 05/21 18:00
+          <p>Nicole Guerriero and Barbara Kruger talk about clawing to success through white walls, specifically white 
+            male walls. The two speakers will be calling audience members onto the stage to workshop personal situations
+            of how to navigate industry.
+          </p></li>
+        <li>LEGACY 05/22 18:00
+          <p>
+            Choi, Guerriero, Knowles, and Kruger join forces one last time for a candid talk of what the future looks like 
+            for "othered" creatives. They ponder on how technology should be used for equality advancement in the future,
+            and preach the necessity of population in order to make long-lasting change within industry specifically.
+          </p></li>
+    </ul> 
+    <li>WHEN &amp; WHERE 
+    
+      <p>
+        The starting on May 20th, 2021 The KITCHEN will be taking over Manhattan's <strong>Terminal 5</strong> for an entire three days. 
+        Join us Thursday thru Saturday, every day from 10am to 7pm for workshops, artisan sales, meet&apos;n&apos;greets, and more. </p>
+      <p>Terminal 5<br> 6 10 W 56th St<br> New York, NY, USA<br> 10019
+    
+    
+        
+        
+    
+        
+        
+      </p></li>
+    <li>CONTACT 
+      <form>
+      <label for="name">NAME</label><br>
+      <input type="text" id="name" name="name"><br>
+      <label for="email">EMAIL</label><br> 
+      <input type="text" id="email" name="email"><br> 
```

Comment:
There's a separate `input` type just for emails 👍🏻
```suggestion
      <input type="email" id="email" name="email"><br> 
```